sources/core-sw/src/compression/include/deflate_slow_matcher.h

```diff
@@ -24,4 +24,11 @@ deflate_match_t get_lazy_best_match(const deflate_hash_table_t *const hash_table
 }
 #endif
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+    deflate_match_t get_lazy_best_match(const deflate_hash_table_t * const hash_table_ptr, const uint8_t * const lower_bound_ptr, const uint8_t * const string_ptr, const uint8_t * const upper_bound_ptr);
+#ifdef __cplusplus
+} // extern "C"
+#endif
```

Comment:
I've looked into changes more carefully and actually, this seems to be duplication of L18-L22 of the same file.